shortfin/shortfin/llm/config.py

```diff
@@ -115,6 +116,14 @@ def max_decode_batch_size(self) -> int:
     def max_batch_size(self):
         return max(self.max_prefill_batch_size, self.max_decode_batch_size)
 
+    @staticmethod
+    def load_json(path):
+        f = open(path)
+        j = json.load(f)
+        return ModelParams(
+            attn_dtype=HalElementType.FLOAT_16, **j)
```

Comment:
Always f16?

Reply:
It is for now, overall we should figure out how to get this value out of the model construction (not possible just yet).